packages/Browser/Number/useRandomInteger/index.ts

```diff
@@ -6,9 +6,9 @@
  */
 export default function useRandomInteger(min: number, max: number) {
   if (min > max) {
-    const swap = min
-    min = max
-    max = swap
+		min = min ^ max
+		max = min ^ max
+		min = min ^ max
```

Comment:
I think this way of writing does not seem to give a good understanding of the meaning.

Reply:
Yes, I think it is unnecessary about its perfomance. Just a temp various but reducing readability. 
But thank you again for your PR！